README.md

```diff
@@ -13,7 +13,7 @@ Documentation is here: https://www.pilosa.com/docs/pdk/
 
 ## Installing
 
-We assume you are on a UNIX-like operating system. Otherwise adapt the following instructions for your platform.
+We assume you are on a UNIX-like operating system. Otherwise adapt the following instructions for your platform. We further assume that you have a Go development environment set up. You should have $GOPATH/bin on your $PATH for access to installed binaries.
```

Comment:
Perhaps link to Go's Getting Started guide, ie: [Go development environment](https://golang.org/doc/install)